frontend/src/components/Footer/Footer.js

```diff
@@ -22,6 +21,7 @@ const Footer = () => {
           >
             <i className="ti-github" aria-hidden="true"></i>
           </a>
+          <hr className={styles.divider}></hr>
```

Comment:
```suggestion
          <hr className={styles.divider} />
```

self closing tag로 사용 안하신 이유가 있을까요?

Reply:
저는 이제껏 가상요소(before, after)를 이용해서 구분선을 개발하였는데요.
hr 태그가 있군요. 소피아 덕분에 하나 배워갑니다. Very Good!!

Reply:
self closing 관련해서는 바로 아래 [코멘트](https://github.com/woowacourse/service-apply/pull/607#discussion_r952815126) 확인해주세요~